src/test/scala/uk/gov/nationalarchives/tdr/api/service/ValidateFileMetadataServiceSpec.scala

```diff
@@ -112,26 +114,29 @@ class ValidateFileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with
     testSetUp.stubMockResponses(existingMetadataRows)
 
     val service = testSetUp.service
+
     val response = service.validateAdditionalMetadata(fileIds, Set("ClosureType", "description")).futureValue
 
     response.size shouldBe 4
 
+    val convertedResponse = convertFilestatusRowToAddFileStatusInput(response)
```

Comment:
I would rename this val to: `expectedAddFileStatusInput` Make it clearer what it actual is, regardless of how it is derived for testing.

Reply:
Can you rename this val: `expectedAddFileStatusInput`. Thanks